api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -269,11 +273,20 @@ private User initializeUserIfNeeded() {
         return user;
 
       case ERROR:
-        log.log(Level.SEVERE, String.format(
-            "free tier project %s failed to be created", user.getFreeTierBillingProjectName()));
-        user.setFreeTierBillingProjectStatus(status);
-        return userDao.save(user);
-
+        int retries = Optional.ofNullable(user.getBillingProjectRetries()).orElse(0);
+        if (retries <= 2) {
```

Comment:
If you moved `2` to a configuration value, we would be able to use it in the test, and perhaps adjust it more easily if this is problematic.

Reply:
Sure! I can do that.